testdata/p4tc_samples_outputs/calculator.template

```diff
@@ -34,9 +34,9 @@ $TC p4template create table/calculator/MainControlImpl/calculate \
 	act name calculator/MainControlImpl/operation_xor \
 	act name calculator/MainControlImpl/operation_drop
 $TC p4template update table/calculator/MainControlImpl/calculate default_miss_action permissions 0x1024 action calculator/MainControlImpl/operation_drop
-$TC p4template create table/calculator/MainControlImpl/calculate entry op 2b action calculator/MainControlImpl/operation_add permissions 0x1024
-$TC p4template create table/calculator/MainControlImpl/calculate entry op 2d action calculator/MainControlImpl/operation_sub permissions 0x1024
-$TC p4template create table/calculator/MainControlImpl/calculate entry op 26 action calculator/MainControlImpl/operation_and permissions 0x1024
-$TC p4template create table/calculator/MainControlImpl/calculate entry op 7c action calculator/MainControlImpl/operation_or permissions 0x1024
-$TC p4template create table/calculator/MainControlImpl/calculate entry op 5e action calculator/MainControlImpl/operation_xor permissions 0x1024
+$TC p4template create table/calculator/MainControlImpl/calculate entry op 2b action calculator/MainControlImpl/operation_add() permissions 0x1024
```

Comment:
Similar error as with the DPDK back end here?

Reply:
I assumed these were just log messages, are they not?

The entry action may vary based on the parameters of the action call, so just printing the action name here doesn't seem like enough information anyway.

For example: https://github.com/p4lang/p4c/pull/4354/files#diff-26cbcf98a38e0ae0e3e2de079be734bbda20fee14fb5c39cd6517e7c34d68d85R22-R24

Reply:
@komaljai

Reply:
Oh, this is a bash script. @komaljai Should the `tc` commands be updated to somehow take the action param values for each entry action, or is the action name enough?

Reply:
It looks like a command to me so in any case the formatting seems wrong here.

Reply:
I have updated the `p4tc` backend to continue using just the action's name here.

Reply:
> Oh, this is a bash script. @komaljai @usha1830 Should the `tc` commands be updated to somehow take the action param values for each entry action, or is the action name enough?

I presume we only expect the action name here. @vbnogueira @komaljai can comment further.

Reply:
> > Oh, this is a bash script. @komaljai @usha1830 Should the `tc` commands be updated to somehow take the action param values for each entry action, or is the action name enough?
> 
> I presume we only expect the action name here. @vbnogueira @komaljai can comment further.

So, @kfcripps is right. It should pass the parameters also.
The format would be the following (imagine that operation_add had two parameters "a" and "b" in the calc example). So something like:

```
action operation_add(bit<32> a, bit<32> b) {
        send_back(a + b);
}
```
 The const entry declaration would look like:
 
 ```
 const entries = {
             0x2b : operation_add(1 + 42);
            ...
}
 ```
 
 And the template script should generate:
 ```
 $TC p4template create table/calc/MainControlImpl/calculate entry op 0x2b permissions 0x1024 action calc/MainControlImpl/operation_test param a 1 param b 42
 ```